spec/models/info_request/state/calculator_spec.rb

```diff
@@ -253,4 +253,53 @@
     end
   end
 
+  describe '#summarised_phase' do
+    # A summarised version of #phase, grouping the phases down into 3 groups
+    def summarised_phase(cached_value_ok=false)
```

Comment:
An aide-memoire I left in by mistake, removed in b14ec56fb9c603cdd6e7f65c770d1b7c09ec9129